controllers/preflightvalidation_reconciler.go

```diff
@@ -120,7 +120,7 @@ func (r *PreflightValidationReconciler) runPreflightValidation(ctx context.Conte
 	for _, module := range modulesToCheck {
 		log.Info("start module preflight validation", "name", module.Name)
 
-		verified, message := r.preflight.PreflightUpgradeCheck(ctx, &module, pv.Spec.KernelVersion)
+		verified, message := r.preflight.PreflightUpgradeCheck(ctx, pv, &module, pv.Spec.KernelVersion)
```

Comment:
Why do we need both `pv` and `pv.Spec.KernelVersion`?

Reply:
this is the way it was added to the upstream. We can probably remove it, but it is better to do it in synchronized PRs both to upstream and downastream

Reply:
`pv.Spec.KernelVersion` isn't present u/s.
See https://github.com/kubernetes-sigs/kernel-module-management/blob/2642a7db8531314c6ae4f2504cd1e4c0dc69d3aa/controllers/preflightvalidation_reconciler.go#L123